packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -665,6 +665,12 @@ export default {
     },
   },
 
+  created () {
+    if (this.value && this.outputFormat === 'text') {
+      this.internalInputValue = this.value.replace(/\n/g, '<br>');
+    }
+  },
```

Comment:
This only needs to be done on created? Couldn't this potentially be set after the message input is initialized as well?

Reply:
I think we should be fine with doing this only on creation. When the input is already initialized, it means user is writing or on the same page; when adding a line break there, it is correctly calculated as a `\n`. So I think we should be fine

https://github.com/user-attachments/assets/1be87a5a-7d17-4aa9-bee9-0c14eaf1c73e

I agree that this is a bit tricky, but I couldn't find anything better without modifying the backend.

Reply:
yeah no prob, the backend definitely needs a lot of changes to work smoothly with this.